.env.deployment.example

```diff
@@ -0,0 +1,42 @@
+# Used by the multi-chain deployment script
+
+# General
+export MAX_SEGMENT_COUNT="THE_MAX_SEGMENT_COUNT"
+export MNEMONIC="YOUR_MNEMONIC"
+
+# RPC URLs
+export ARBITRUM_RPC_URL="YOUR_RPC_URL"
+export ARBITRUM_SEPOLIA_RPC_URL="YOUR_RPC_URL"
+export AVALANCHE_RPC_URL="YOUR_RPC_URL"
+export BASE_RPC_URL="YOUR_RPC_URL"
+export BSC_RPC_URL="YOUR_RPC_URL"
+export GNOSIS_RPC_URL="YOUR_RPC_URL"
+export MAINNET_RPC_URL="YOUR_RPC_URL"
+export OPTIMISM_RPC_URL="YOUR_RPC_URL"
+export POLYGON_RPC_URL="YOUR_RPC_URL"
+export SCROLL_RPC_URL="YOUR_RPC_URL"
+export SEPOLIA_RPC_URL="YOUR_RPC_URL"
+
+# Etherscan API keys
+export ARBISCAN_API_KEY="YOUR_API_KEY"
+export BASESCAN_API_KEY="YOUR_API_KEY"
+export BSCSCAN_API_KEY="YOUR_API_KEY"
+export ETHERSCAN_API_KEY="YOUR_API_KEY"
+export GNOSISSCAN_API_KEY="YOUR_API_KEY"
+export OPTIMISTIC_API_KEY="YOUR_API_KEY"
+export POLYGONSCAN_API_KEY="YOUR_API_KEY"
+export SCROLLSCAN_API_KEY="YOUR_API_KEY"
+export SNOWTRACE_API_KEY="YOUR_API_KEY"
+
+# Script variables
+export ARBITRUM_ADMIN="YOUR_ADMIN_ADDRESS"
+export ARBITRUM_SEPOLIA_ADMIN="YOUR_ADMIN_ADDRESS"
+export AVALANCHE_ADMIN="YOUR_ADMIN_ADDRESS"
+export BASE_ADMIN="YOUR_ADMIN_ADDRESS"
+export BSC_ADMIN="YOUR_ADMIN_ADDRESS"
+export GNOSIS_ADMIN="YOUR_ADMIN_ADDRESS"
+export MAINNET_ADMIN="YOUR_ADMIN_ADDRESS"
+export OPTIMISM_ADMIN="YOUR_ADMIN_ADDRESS"
+export POLYGON_ADMIN="YOUR_ADMIN_ADDRESS"
+export SCROLL_ADMIN="YOUR_ADMIN_ADDRESS"
+export SEPOLIA_ADMIN="YOUR_ADMIN_ADDRESS"
```

Comment:
We can keep these values directly in version control because the admin addresses are public:

https://docs.sablier.com/concepts/governance#multisig-addresses